user/views.py

```diff
@@ -1,3 +1,95 @@
-from django.shortcuts import render
+import json
+import boto3
+import uuid
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from .models     import User
+from .utils      import id_auth
+from my_settings import (
+                            AWS_ACCESS_KEY_ID,
+                            AWS_SECRET_ACCESS_KEY,
+                        )
+
+
+class ProfileView(View):
+
+    @id_auth
+    def post(self, request):
+        data = json.loads(request.body)
+
+        user = request.user
+        name           = data['name']
+        nickname       = data['nickname']
+
+        try:
+            User.objects.filter(id = user.id).update(
+                name          = name,
+                nickname      = nickname,
+            )
+            
+            return JsonResponse({'message': "SUCCESS"}, status=201)
+
+        except KeyError as e:
+            return JsonResponse({'message': f'KEY_ERROR: =>  {e}'}, status=400)
+
+        except ValueError as e:
+            return JsonResponse({'message': f'VALUE_ERROR: =>  {e}'}, status=400)
+
+
+class ProfileDataView(View):
+
+    @id_auth
+    def get(self,request):
+        user = request.user
+
+        try:
+            result = {
+                "name"     : User.objects.get(id = user.id).name,
+                "nickname" : User.objects.get(id = user.id).nickname,
+                "image"    : User.objects.get(id = user.id).profile_image,
+                "email"    : User.objects.get(id = user.id).email,
+                "phone"    : User.objects.get(id = user.id).number,
+            }
```

Comment:
민철님 여기서 User 객체를 매번 가져오는 이유가 무엇인가요??
```suggestion
        try:
            result = {
                "name"     : User.objects.get(id = user.id).name,
                "nickname" : User.objects.get(id = user.id).nickname,
                "image"    : User.objects.get(id = user.id).profile_image,
                "email"    : User.objects.get(id = user.id).email,
                "phone"    : User.objects.get(id = user.id).number,
            }
```